mscore/metaedit.ui

```diff
@@ -100,77 +199,134 @@
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
-     <widget class="QWidget" name="scrollWidget">
+     <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
-        <width>646</width>
-        <height>238</height>
+        <width>660</width>
+        <height>297</height>
        </rect>
       </property>
-      <layout class="QGridLayout" name="gridLayout">
-       <property name="verticalSpacing">
-        <number>1</number>
+      <layout class="QGridLayout" name="scrollAreaLayout">
+       <property name="spacing">
+        <number>3</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
-    <layout class="QHBoxLayout" name="horizontalLayout_2">
+    <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="newButton">
+       <property name="cursor">
+        <cursorShape>PointingHandCursor</cursorShape>
+       </property>
+       <property name="toolTip">
+        <string>Add a tag to this score.</string>
+       </property>
        <property name="text">
-        <string notr="true">+</string>
+        <string notr="true">New...</string>
```

Comment:
`+` had the advantage of not needing a translation. Now the string does need a translation, but translation is disabled.